vmm/src/config.rs

```diff
@@ -1197,6 +1210,35 @@ impl SgxEpcConfig {
     }
 }
 
+#[derive(Clone, Debug, PartialEq, Deserialize, Serialize, Default)]
+pub struct NumaConfig {
+    #[serde(default)]
+    pub id: u64,
+    #[serde(default)]
+    pub cpus: Option<Vec<u64>>,
+}
+
+impl NumaConfig {
+    pub const SYNTAX: &'static str = "Settings related to a given NUMA node \
+        \"id=<node_id>,cpus=<cpus_id>\"";
```

Comment:
Please extend the syntax to make clear this is a comma separated list of CPU ids.

Reply:
Actually, looking at the code it looks more complex than that... :)

Reply:
That's where I need feedback. I chose to go with following syntax `cpus=3-6:9:10-12`, meaning CPUs 3,4,5,6,9,10,11,12 will be assigned to the given NUMA node.
Do you have any other idea on how to make this less ugly?
That being said, it is ugly but we don't expect lots of users to use this syntax anyway.

Reply:
Do we use colons anywhere else for separating a list? Feels a little bit weird to me. Especially if you're happy with the zipped syntax I suggest in #1657

Reply:
To be clear i'm thinking commas and dashes.